rentable.rb

```diff
@@ -0,0 +1,20 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  @rentals = []
```

Comment:
- [ ] Do we really need this `@rentals` item here? Kindly remove it or elaborate on how it is useful for the next reviewer. But I don't think we need it, so I think we should remove it to avoid unneeded code.

  We can find all the rentals in the respective book and person objects that were used to create a rental.